desc/plotting.py

```diff
@@ -685,7 +684,9 @@ def plot_2d(
         * ``xlabel_fontsize``: float, fontsize of the xlabel
         * ``ylabel_fontsize``: float, fontsize of the ylabel
         * ``cmap``: str, matplotlib colormap scheme to use, passed to ax.contourf
-        * ``levels``: int or array-like, passed to contourf
+        * ``levels``: int or array-like, passed to contourf.
+          If ``name``="|F|_normalized" and ``log``==True, default is
+          np.logspace(-6, 0, 7). Pass None to use actual data min/max.
```

Comment:
```suggestion
        * ``levels``: int or array-like, passed to contourf.
          If ``name="|F|_normalized"`` and ``log=True``, default is
          ``np.logspace(-6, 0, 7)``. Otherwise the default (``None``) uses the min/max
          values of the data.
```